apps/dev-tools/vite.config.ts

```diff
@@ -0,0 +1,12 @@
+import react from '@vitejs/plugin-react'
+import path from 'path'
+import { defineConfig } from 'vite'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+	plugins: [react()],
+	root: path.join(__dirname, 'src'),
+	server: {
+		port: 5420,
```

Comment:
let's put this on a different port. 5420 is the examples app port. we can pick something else!

Reply:
4000 now.